group/nist/suite.go

```diff
@@ -24,9 +24,8 @@ func (s *Suite128) Hash() hash.Hash {
 	return sha256.New()
 }
 
-// SHA3/SHAKE128 Sponge Cipher
-func (s *Suite128) Cipher(key []byte, options ...interface{}) kyber.Cipher {
-	return sha3.NewShakeCipher128(key, options...)
+func (s *Suite128) XOF(key []byte) kyber.XOF {
```

Comment:
And one more:

`NewBlakeSHA256P256`

?

Reply:
Fixed.